collector/runtime-benchmarks/async/src/main.rs

```diff
@@ -0,0 +1,47 @@
+use benchlib::benchmark::run_benchmark_group;
+use std::time::Instant;
+use tokio::runtime::Runtime;
+use std::fs::File;
+use std::io::{self, BufRead, BufReader};
+use flate2::read::GzDecoder;
+
+async fn total_char_count(reader: BufReader<GzDecoder<File>>,x: &mut usize) {
+    for line in reader.lines() {
+        *x += line_char_count(line.expect("invalid character")).await;
+    }
+}
+
+async fn line_char_count(line: String) -> usize {
+    let line_count = line.chars().count();
+    line_count
+}
+
+async fn async_operation() -> (usize, u128) {
+    let start_time = Instant::now();
+
+    let file = File::open("./collector/runtime-benchmarks/data/sherlock.txt.gz").expect("can't read a file");
```

Comment:
Let's use relative paths and inline the file into the binary to avoid I/O during the benchmark ( see https://github.com/rust-lang/rustc-perf/blob/c1f870e4f1089ae12360e034657256382fe2ddcf/collector/runtime-benchmarks/compression/src/main.rs#L8).